warehouse/integrations/vulnerabilities/utils.py

```diff
@@ -101,19 +101,26 @@ def _analyze_vulnerability(request, vulnerability_report, origin, metrics):
         )
         raise
 
+    found_releases = False  # by now, we don't have any release found
+
     for version in report.versions:
         try:
             release = _get_release(request, project, version)
+            found_releases = True  # at least one release found
         except NoResultFound:
             metrics.increment(
                 "warehouse.vulnerabilities.error.release_not_found",
                 tags=[f"origin:{origin}"],
             )
```

Comment:
Is this part necessary? Because I think that is making the tests to fail.

Reply:
Yes, it's still necessary. The tests are failing because the behavior has changed here but the tests haven't been updated.

Reply:
Got it. As I understand, we should change `tests/unit/integration/vulnerabilities/test_utils.py`, right?